From 19f79eefac18b6cb854d7038135c2173818639fb Mon Sep 17 00:00:00 2001
From: Nicolas Ferre <nicolas.ferre@microchip.com>
Date: Mon, 2 Oct 2017 16:08:22 +0200
Subject: [PATCH 434/471] ARM: at91: sama5_defconfig: remove PERF and KALLSYMS

For performance reasons, remove some symbols and perf events to the default
configuration.

Signed-off-by: Nicolas Ferre <nicolas.ferre@microchip.com>
---
 arch/arm/configs/sama5_defconfig | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/arch/arm/configs/sama5_defconfig b/arch/arm/configs/sama5_defconfig
index 62737dd35bcc..3cedde459fd1 100644
--- a/arch/arm/configs/sama5_defconfig
+++ b/arch/arm/configs/sama5_defconfig
@@ -12,12 +12,11 @@ CONFIG_RT_GROUP_SCHED=y
 CONFIG_CGROUP_FREEZER=y
 CONFIG_CGROUP_DEVICE=y
 CONFIG_CGROUP_CPUACCT=y
-CONFIG_CGROUP_PERF=y
 CONFIG_NAMESPACES=y
 CONFIG_USER_NS=y
 CONFIG_BLK_DEV_INITRD=y
+# CONFIG_KALLSYMS is not set
 CONFIG_EMBEDDED=y
-CONFIG_PERF_EVENTS=y
 CONFIG_SLAB=y
 CONFIG_ARCH_AT91=y
 CONFIG_SOC_SAMA5D2=y
-- 
2.34.1

